### PR TITLE
Add dnt support in s2s ( Proto changes)

### PR DESCRIPTION
**Pull request by shiralal**
Added dnt_phrases parameter in translation_config.

### CODE REVIEW

File: riva/proto/riva_nmt.proto
Comment (rmittal-github) [inline riva/proto/riva_nmt.proto:82]: will need to add a description of this field here

**Comment by rmittal-github** [comment]
@shiralal please rebase on top of `release/2.17.0` branch, since it is targeted for upcoming release.

File: riva/proto/riva_nmt.proto
Comment (shiralal) [reply riva/proto/riva_nmt.proto:82]: Its done

**Comment by shiralal** [comment]
> @shiralal please rebase on top of `release/2.17.0` branch, since it is targeted for upcoming release.

rebased on `release/2.17.0` branch

File: riva/proto/riva_nmt.proto
Comment (rmittal-github) [reply riva/proto/riva_nmt.proto:82]: will be good to add a more descriptive comment like here https://github.com/nvidia-riva/common/blob/main/riva/proto/riva_asr.proto#L269

**Review by rmittal-github — APPROVED** [review]
LGTM, will wait for CI to complete before merge

**Pull request merged**
